### ISSUE TITLE
Compiler error while building for android

### ISSUE THREAD

**Issue by Tennboard** [2d-inc/Flare-Flutter#78]
Hello, i have deleted some .flr files from project which i dont use anymore commited changes to github and then try to build app trough codemagic and it fails on android build with error: 

(if i build localy on computer it builds just fine, but trough codemagic CI it breaks...)
```
== Building for Android ==

== /usr/local/bin/flutter build apk --debug ==
Initializing gradle...                                              2.1s
Resolving dependencies...                                          50.8s
Running Gradle task 'assembleDebug'...                          


Compiler message:

file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/flare_flutter-1.4.0/lib/flare_cache_asset.dart:20:59: Error: The argument type 'Null Function(FlutterActor)' can't be assigned to the parameter type 'FutureOr<dynamic> Function(Future<FlutterActor>)'.

 - 'FlutterActor' is from 'package:flare_flutter/flare.dart' ('file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/flare_flutter-1.4.0/lib/flare.dart').

 - 'FutureOr' is from 'dart:async'.

 - 'Future' is from 'dart:async'.

Try changing the type of the parameter, or casting the argument to 'FutureOr<dynamic> Function(Future<FlutterActor>)'.

        compute(FlutterActor.loadFromByteData, data).then((FlutterActor actor) {

                                                          ^

Compiler failed on lib/main.dart
Running Gradle task 'assembleDebug'...                             24.9s
Gradle task assembleDebug failed with exit code 1
```

Any ideas why that happens?

**Comment by ScrivNetwork** [comment]
same error

**Comment by grandpa-guru** [comment]
Duplicate of https://github.com/2d-inc/Flare-Flutter/issues/77.

**Comment by luigi-rosso** [comment]
We're working through some breaking changes between Flutter stable and master/dev. If you need a build for flutter stable right now, please point your pub to 1.3.13 as follows:

flare_flutter: 1.3.13

Note there's no ^ before the version number.

**Comment by luigi-rosso** [comment]
We have a new system in place for handling these breaking changes. Please update your pubspec to ```flare_flutter: ^1.5.0``` and take a look at our updated [README](https://github.com/2d-inc/Flare-Flutter/).

**Comment by Tennboard** [comment]
Its fixed now with 1.5.0, thanks :)

**Comment by grandpa-guru** [comment]
@luigi-rosso Also, change the Flutter channel to stable.
